.github/workflows/ci-comment.yml

```diff
@@ -55,4 +55,25 @@ jobs:
               issue_number: issue_number,
               body: `The commit ${pr_sha} (as a parent of ${merge_sha}) contains errors. Please inspect the [Run Summary](${context.payload.workflow_run.html_url}) for details.`
             });
+            await octokit.request(`POST /repos/{owner}/{repo}/issues/{issue_number}/labels`, {
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              issue_number: issue_number,
+              labels: [ 'waiting: response from author' ]
+            })
+
+      - name: On Success
+        uses: actions/github-script@v6
+        if: ${{ github.event.workflow_run.conclusion != 'failure' }}
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            let fs = require('fs');
+            let issue_number = Number(fs.readFileSync('./pr_number', 'utf8').trim());
```

Comment:
I know this file uses `readFileSync` elsewhere, but as a general best practice we should be using async methods.  Also, we should be using `const` rather than `let` as another best practice (doesn't matter much for a tiny script like this, but a generally good habit to be in when we are writing TS code).

Reply:
This isn't TS, but your point is valid.